en/developer_guide/custom_content_attribute_display.rst

```diff
@@ -0,0 +1,68 @@
+Customize Content Attribute Display
+===================================
+
+Context
+-------
+
+In Open Orchestra the ``Content`` are composed by a set of base attributes and a collection of
+``ContentAttribute`` (See `content`_ documentation for more information). The ``ContentAttribute``
+are configured by the `content type`_.
+
+In order to have minimum constraints on contents, Open Orchestra allows you to store any type
+of ``value`` in the ``ContentAttribute``.
+
+Sometimes, there is no easy way to render the value to the backoffice user.
+A custom ``ContentAttribute`` could be an array, which cannot be displayed directly in javascript.
```

Comment:
I clearly see why you're talking about javascript, but I'm not sure an external developper will understand where javascript comes into the problem. Maybe you could talk about the datatable plugin.